docs/contributors/code/release.md

```diff
@@ -154,9 +164,25 @@ _If_ however, the previous release was an **RC** (e.g. `X.Y.0-rc.1`) you will ne
 
 To do this, when running the Workflow, select the appropriate `release/` branch from the `Use workflow from` dropdown (e.g. `release/12.5`) and specify `stable` in the text input field.
 
-Please note you **cannot create point releases for previous stable releases once a more recent stable release has been published** as this would require significant changes to how we upload plugin versions to the WP.org plugin SVN repo). Always check the latest release version before you proceed.
+Please note you **cannot create point releases for previous stable releases once a more recent stable release has been published** as this would require significant changes to how we upload plugin versions to the WP.org plugin SVN repo). Always check the latest release version before you proceed (see [this Issue](https://github.com/WordPress/gutenberg/issues/33277#issuecomment-876289457) for more information).
+
+#### Troubleshooting
+
+> The release draft was created but it was empty/contained an error message
+
+If you forget to assign the correct Milestone to your cherry picked PR then the release tool may error or your changelog may be empty.
```

Comment:
In general the PRs without the proper milestone will be omitted. So while this can be more obvious  when we forget to do this for all the PRs(which are not many usually), it can still be the case of missing entries if for example we cherry picked three PRs but fixed the milestone for two of them. We would still need to check if the entries are the same number with the cherry picked PRs.